model/src/conditions/inline-condition-model.js

```diff
@@ -194,6 +194,29 @@ export class ConditionsModel {
     }
     return []
   }
+
+  toJSON () {
```

Comment:
this isn't a returning JSON. do you mean to toObject? or to wrap in the return in `JSON.stringify(..)`?

Reply:
No, this is part of the JSON.stringify functionality. In order to customise the serialisation you define a toJSON method returning the object you want to be serialised: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#toJSON_behavior

Reply:
oh, that's interesting!